checks/normandy/recipes_signatures.py

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
I think the filename of this should be `recipe_signatures.py`, instead of `recipes_signatures.py`. That is, only "signatures" is plural, since English is weird about plurals and adjectives.